### ISSUE TITLE
Fix an object conversion issue

### ISSUE THREAD

**Issue by ddobrin** [networknt/swagger-bundler#3]
Keys are not always strings, and conversion was failing.

**Comment by whiskeysierra** [comment]
Does the latest docker image contain this fix? Because I see this

```
java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.String
	at com.networknt.swagger.Bundler.resolveMap(Bundler.java:184)
	at com.networknt.swagger.Bundler.resolveMap(Bundler.java:193)
	at com.networknt.swagger.Bundler.resolveMap(Bundler.java:193)
	at com.networknt.swagger.Bundler.resolveMap(Bundler.java:193)
	at com.networknt.swagger.Bundler.main(Bundler.java:40)
```

**Comment by stevehu** [comment]
@whiskeysierra Sorry. This utility is used by one of our customers who has too many swagger files shared between APIs. They are using this utility as Java command line tool with "java -jar ..." so I didn't publish the docker image after the fix. I have just published version 0.1.2 to docker hub minutes ago and the new version will have the issue fixed. Thanks a lot for bringing it up. 

https://hub.docker.com/r/networknt/swagger-bundler/tags/
